### ISSUE TITLE
LRC files being saved without "." for file extension

### ISSUE THREAD

**Issue by stevecole90099** [justin025/onthespot#51]
On Windows 11 24h2.
Version 1.0.1

App is downloading and saving lyrics to a file as I've set in the app settings but the file is being saved as an unknown file because the "." that separates the filename and extension is missing. 
Edit:
I didn't notice it before but it looks like it is also missing some of the name also. So far from the other files I've looked at it looks like it is missing the last three letters of the name and the period for the file extension.

Example track: Sunset by The Midnight

Files in folder:
"2. Sunlrc"
"2. Sunset.mp3"

From log file:
```
[2024-11-08 18:36:10,361 :: search :: search.py -> 36:  get_search_results() :: INFO] -> Search clicked with value term the sunset
[2024-11-08 18:36:10,408 :: api.spotify :: api\spotify.py -> 392:spotify_get_search_results() :: INFO] -> Get search result for term 'the sunset'
[2024-11-08 18:36:16,399 :: parse_item :: parse_item.py -> 66:       parsingworker() :: INFO] -> Parsing: {'item_url': 'https://open.spotify.com/track/6mB9A9YLbY4jxpKX5EYAnT', 'item_service': 'spotify', 'item_type': 'track', 'item_id': '6mB9A9YLbY4jxpKX5EYAnT'}
[2024-11-08 18:36:18,956 :: api.spotify :: api\spotify.py -> 255:  spotify_get_lyrics() :: INFO] -> ['[ar:The Midnight]', '[al:Endless Summer]', '[ti:Sunset]', '[au:Tim McEwan; Tyler Lyle]', '[by:MusixMatch]', '[length:05:27]\n', "[00:19.59] I just feel like, I've been stuck here my whole life", '[00:26.42] There must be something else out there, waiting, you know?', '[00:31.56] Just for us', '[00:35.78] We could run away', '[00:39.15] Just leave, never look back', "[00:46.16] I don't even care where we go", '[00:48.50] Just far away from here', "[00:51.87] I'll come with you", '[00:53.03] (One, two, three, four)', '[00:56.10] ♪', '[01:10.90] Faked our deaths, lit a match', '[01:15.27] Closed the door, waited for the flash', '[01:19.25] Greyhound Station, we paid in cash', "[01:23.32] Miss Lazarus, we're coming back", "[01:28.40] They say it's darkest before the dawn", "[01:32.85] We've been in this town for far too long", "[01:37.01] They say it's darkest before the dawn", "[01:40.77] We're moving on, we're moving on, we're moving on", '[01:45.21] Sunset, no regrets', '[01:49.26] First chance, last dance, stuck in the middle', '[01:53.28] Even bad girls know good love', '[01:57.99] You taste so sweet it hurts a little', '[02:18.37] Make our home, where we stand', '[02:22.80] One suitcase and half a plan', '[02:26.73] Follow the wolves and sing along', '[02:31.21] Past the voices to the song', "[02:36.22] They say it's darkest before the dawn", "[02:40.29] We've been in this town for far too long", "[02:44.60] They say it's darkest before the dawn", "[02:48.35] We're moving on, we're moving on, we're moving on", '[02:52.75] Sunsets, no regrets', '[02:56.95] First chance, last dance, stuck in the middle', '[03:00.45] Even bad girls know good love', '[03:05.38] You taste so sweet it hurts a little', '[03:58.87] ♪', '[04:16.82] Sunsets, no regrets', '[04:21.12] First chance, last dance, stuck in the middle', '[04:24.73] Even bad girls know good love', '[04:29.49] You taste so sweet it hurts a little', '[04:33.94] Sunsets, no regrets', '[04:38.10] First chance, last dance, stuck in the middle', '[04:41.72] Even bad girls know good love', '[04:46.33] You taste so sweet it hurts a little', '[04:50.59] ']
[2024-11-08 18:36:24,140 :: post_download :: post_download.py -> 213: set_music_thumbnail() :: INFO] -> Fetching item thumbnail
[2024-11-08 18:36:25,041 :: downloader :: downloader.py -> 340:                 run() :: INFO] -> Item Successfully Downloaded
```

**Comment by justin025** [comment]
should be fixed by this commit exe is at the bottom of the page, sorry about that. https://github.com/justin025/onthespot/actions/runs/11757275003

**Comment by stevecole90099** [comment]
> should be fixed by this commit exe is at the bottom of the page, sorry about that. https://github.com/justin025/onthespot/actions/runs/11757275003

This exe solves the issue. Thanks for the quick fix.

**Comment by stevecole90099** [comment]
> should be fixed by this commit exe is at the bottom of the page, sorry about that. https://github.com/justin025/onthespot/actions/runs/11757275003

I've noticed a new issue with the EXE from this commit. Not sure if it is just with this version or not so I'm not sure if I should open a new issue.

The issue appears to be with albums that have multiple discs, the software will report that the tracks from disc two already exist if disc one is already downloaded and the tracks share a track number. I was able to get around the issue by changing the default track name to no longer start with the track number ( {album_artist}\{album} [{year}]\{name} ), so I think the issue might be with how the app looks for existing files.

Effected Album: Ultimate Aaliyah by Aaliyah
